newdle/api.py

```diff
@@ -291,6 +291,25 @@ def get_newdles_participating():
     return NewdleParticipantSchema(many=True).jsonify(newdle)
 
 
+@api.route('/newdle/<code>/create-event', methods=('POST',))
+def create_event(code):
+    newdle = Newdle.query.filter_by(code=code).first_or_404(
+        'Specified newdle does not exist'
+    )
+    url = current_app.config['INDICO_URL']
+    res = requests.post(
+        url,
+        data={
+            'title': newdle.title,
+            'start_dt': newdle.final_dt,
+            # Newdle accepts the duration in minutes.
+            'duration': (newdle.duration.seconds // 60) % 60,
```

Comment:
`int(newdle.duration.total_seconds()) // 60` and I the modulus isn't needed - we can have meetings longer than 60 minutes! ATM this actually returns 0 if it's exactly 1 hour...

Reply:
Didn't realize that before, thanks.